src/sagas/index.js

```diff
@@ -0,0 +1,74 @@
+import axios from 'axios';
+import { call, put, takeEvery, all } from 'redux-saga/effects'
+import trendGifsActions from '../actions/trendGifs';
+import searchGifsActions from '../actions/searchGifs';
+
+const { TREND_GIFS_LOAD } = trendGifsActions.types;
+const { fetchedTrendGifs, failedFetchTrendGifs } = trendGifsActions.creators;
+const { SEARCH_GIFS_LOAD } = searchGifsActions.types;
+const { fetchedSearchGifs, failedFetchSearchGifs } = searchGifsActions.creators;
+
+const giphyApiUrl = 'https://api.giphy.com/v1';
+const giphyApiKey = 'iyoTSR6E7wC6lx0XhXApQqHQxbVIfp8v';
```

Comment:
Use .env variables to save your api keys and do not expose them versioning them. You can provide an .env.example file expressing its usage.

https://www.npmjs.com/package/dotenv